resources/connection.rb

```diff
@@ -17,11 +17,13 @@
 # limitations under the License.
 #
 
+unified_mode true
+
 default_action :create
 
 property :service_name, String, name_property: true
-property :connect, required: true
-property :accept, required: true
+property :connect, String, required: true
+property :accept, String, required: true
```

Comment:
These actually should probably be both string or integer `[String, Integer]` then the test cookbooks don't need to be updated with quotes when it's just a port number.